script/src/verify.rs

```diff
@@ -566,8 +579,10 @@ impl<DL: CellDataProvider + HeaderProvider + ExtensionProvider + Send + Sync + C
             ScriptHashType::Type => {
```

Comment:
Do we need to restore the on condition of data1, which was not enabled before 2021 was enable?